projects.yaml

```diff
@@ -442,6 +442,10 @@ projects:
         - *module_if
         - *module_hic_lpc11u35
         - records/board/tiny.yaml
+    lpc11u35_pitaya_link_if:
```

Comment:
It should be higher, between `lpc11u35_nz32_sc151_if` and `lpc11u35_rtl8195am_if`.

Reply:
Resolved per comment